tests/test_order.py

```diff
@@ -1,11 +1,24 @@
+import copy
+from typing import Any, Dict
+from unittest import mock
+
 import pytest
+import requests
 
 from up42 import asset, order
 
 from .fixtures import fixtures_globals as constants
 from .fixtures import fixtures_order
 
```

Comment:
We aligned to fix the coverage. It means we need to revisit all tests in this module, not only the ones affected